### PR TITLE
[fix] add precompile to access list

### PR DESCRIPTION
**Pull request by lightsing**

### CODE REVIEW

**Comment by lightsing** [comment]
\b run testool

**Comment by lightsing** [comment]
\b run testool

File: zkevm-circuits/src/evm_circuit/execution/begin_tx.rs [hunk 1]
```diff
@@ -224,6 +227,17 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
...
 
+        let precompile_warm = array_init::array_init(|_| cb.query_bool());
+        for addr in 1..=PRECOMPILE_COUNT {
+            cb.account_access_list_write(
+                tx_id.expr(),
+                addr.expr(),
+                1.expr(),
+                precompile_warm[addr - 1].expr(),
+                None,
+            ); // rwc_delta += 1
```
Comment (silathdiir) [inline zkevm-circuits/src/evm_circuit/execution/begin_tx.rs:238]:
```suggestion
            ); // rwc_delta += 9
```

**Review by silathdiir — APPROVED** [review]
LGTM

**Comment by silathdiir** [comment]
\b run testool

File: zkevm-circuits/src/evm_circuit/execution/begin_tx.rs [hunk 2]
```diff
@@ -224,6 +227,17 @@ impl<F: Field> ExecutionGadget<F> for BeginTxGadget<F> {
         let gas_left = tx_gas.expr() - intrinsic_gas_cost.expr();
         let sufficient_gas_left = RangeCheckGadget::construct(cb, gas_left.clone());
 
+        let precompile_warm = array_init::array_init(|_| cb.query_bool());
+        for addr in 1..=PRECOMPILE_COUNT {
+            cb.account_access_list_write(
+                tx_id.expr(),
+                addr.expr(),
+                1.expr(),
+                precompile_warm[addr - 1].expr(),
```
Comment (lispc) [inline zkevm-circuits/src/evm_circuit/execution/begin_tx.rs]: why need precompile_warm cells here? I think they must be true?

Comment (lightsing) [reply zkevm-circuits/src/evm_circuit/execution/begin_tx.rs]: does the prev_value always be false?

**Review by lispc — APPROVED** [review]

**Pull request merged**